apps/camera_pipe/camera_pipe.cpp

```diff
@@ -224,10 +229,11 @@ Func color_correct(Func input, ImageParam matrix_3200, ImageParam matrix_7000, P
 }
 
 
-Func apply_curve(Func input, Type result_type, Param<float> gamma, Param<float> contrast) {
+Func apply_curve(Func input, Type result_type, Param<float> gamma, Param<float> contrast, Param<int> blackLevel, Param<int> whiteLevel) {
     // copied from FCam
     Func curve("curve");
 
+#ifdef OLD_CURVE
```

Comment:
Can you just remove the old curve (and the preprocessor directives)? I'd rather just keep the new curve.